src/React.Core/package.json

```diff
@@ -1,19 +1,20 @@
 {
-  "version": "1.0.0",
-  "name": "React.Core",
-  "private": true,
-  "devDependencies": {
-    "babel-core": "^6.3.13",
-    "babel-loader": "^6.2.0",
-    "babel-preset-es2015": "^6.3.13",
-    "babel-preset-stage-0": "^6.3.13",
-    "babel-standalone": "^1.0.0",
-    "gulp": "^3.9.0",
-    "gulp-uglify": "^1.5.1",
-    "json-loader": "^0.5.4",
-    "react": "^0.14.3",
-    "vinyl-named": "^1.1.0",
-    "webpack": "^1.12.9",
-    "webpack-stream": "^2.3.0"
-  }
+	"version": "1.0.0",
+	"name": "React.Core",
+	"private": true,
+	"devDependencies": {
+		"babel-core": "^6.3.13",
+		"babel-loader": "^6.2.0",
+		"babel-preset-es2015": "^6.3.13",
+		"babel-preset-stage-0": "^6.3.13",
+		"babel-standalone": "^1.0.0",
+		"babel-plugin-transform-regenerator": "^6.3.0",
```

Comment:
Hmm, strange. What error do you get if you remove this? In theory it shouldn't be needed at all.